src/main/kotlin/com/dansplugins/fiefs/fief/FiefRepository.kt

```diff
@@ -0,0 +1,78 @@
+package com.dansplugins.fiefs.fief
+
+import com.dansplugins.fiefs.Fiefs
+import java.util.*
+
+/**
+ * Stores all Fiefs in a list and provides methods for accessing and modifying the list.
+ */
+class FiefRepository(private val plugin: Fiefs) {
+    private val fiefs = mutableListOf<Fief>()
+
+    /**
+     * Adds a Fief to the list.
+     * @param fief The Fief to add.
+     */
+    fun addFief(fief: Fief) {
+        fiefs.add(fief)
+    }
+
+    /**
+     * Removes a Fief from the list.
+     * @param fief The Fief to remove.
+     */
+    fun removeFief(fief: Fief) {
+        fiefs.remove(fief)
+    }
+
+    /**
+     * Gets a Fief from the list.
+     * @param name The name of the Fief to get.
+     * @return The Fief with the given name.
+     */
+    fun getFief(name: String): Fief? {
+        for (fief in fiefs) {
```

Comment:
The more functional approach to this adds a good deal of brevity and is clearer in its intent imo:
`return fiefs.single { it.name == name }`

You can also use `.equals(ignoreCase = true)` if you want case insensitive comparisons.

Can multiple fiefs have the same name?